app/assets/stylesheets/print.scss

```diff
@@ -8,6 +8,7 @@
 @import 'govuk_publishing_components/components/print/govspeak-html-publication';
 @import 'govuk_publishing_components/components/print/metadata';
 @import 'govuk_publishing_components/components/print/share-links';
+@import 'govuk_publishing_components/components/print/skip-link';
```

Comment:
This change doesn't look like one that's described in the PR.

Reply:
Good point, have updated the PR description, thanks.